### ISSUE TITLE
JDS Pro 10 always adds /Packages to download URL

### ISSUE THREAD

**Issue by ronnybremer** [brysontyrrell/ODST#9]
I managed to deploy a ODTS instance on a CentOS server. When registering it with JDS, I created a dummy AFS share and enabled HTTP downloads only. I put in "static/share" as the URL, but JDS always tries to download like this:
`http://server.com:5000/static/share/Packages/test.pkg.zip`

I fixed it temporarily by adding a symlink into the share folder, but that is a bad hack. Would it be possible to specify a download directory?

Yours,

Ronny

**Comment by brysontyrrell** [comment]
Hi @ronnybremer,

I've made a commit: 7daf437

Check this out. In the code, I added '/Packages' as an additional fallback for the web app to serve the file if the web server component is not configured to do so.

In the Docker's Nginx config I added an additional alias to the static directory on the server for '/Packages/' so Nginx will handle file downloads when that URI is requested.

In Jamf Pro 10 make sure you don't set any "Context" under the "HTTP/HTTPS" settings. It should work then. You'll see in the client logs `http://<ods-address>/Packages/<package-name>`.
